tests/php/View/SSViewerCacheBlockTest.php

```diff
@@ -151,6 +153,7 @@ public function testBlocksInvalidateOnFlush()
         $this->assertEquals($this->_runtemplate('<% cached %>$Foo<% end_cached %>', ['Foo' => 1]), '1');
 
         // Test without flush
+        Injector::inst()->get(Kernel::class)->boot();
```

Comment:
This needed to be added to get the unit test to pass

It was failing because the cache file in `/tmp/silverstripe-cache-php8.1.9-var-www/www-data/.cache.888b282739860e4b3ef798dd899180cf502a4974` was being deleted by the call to `Director::test('/');`.  This is because `FlushMiddleWare` ... `if (Director::isManifestFlushed()) {` which basically means "doFlush()", was true when it shouldn't have because.

Adding in `Injector::inst()->get(Kernel::class)->boot();` essentially makes it so that the request isn't in a "doFlush" state

I'm honestly not sure how this used to work, possibly in the past the cache file was not being deleted when it should have been.

Reply:
You're probably right, though it's worth noting the a few lines lower within this same unit test we're calling `Injector::inst()->get(Kernel::class)->boot(true);`, so this is the "correct" way to handle this for now, sort of :-)

Reply:
It's super not correct, but I take it your meaning is "this is an existing bug and the workaround being used here is already used below in the same test - fixing the bug is not in scope".
If that's you're meaning, then I can accept that. I've created [a new card](https://github.com/silverstripe/silverstripe-framework/issues/10493) to address the underlying issue in the future.